v1/plugins/logs/plugin.go

```diff
@@ -1129,3 +1199,27 @@ func (p *Plugin) logEvent(event EventV1) error {
 	}).Info("Decision Log")
 	return nil
 }
+
+func (p *Plugin) addMetric(name string) {
```

Comment:
Nice helper, I like how this lets us clean up the `if p.metrics != nil { ... }` sites... I think its name would be a bit more explicit if it was "incrMetric"?

Reply:
Agreed updated it to `incrMetric`

Reply:
Thanks!